src/pydistcheck/cli.py

```diff
@@ -115,16 +115,6 @@ def check(
     print(f"errors found while checking: {len(errors)}")
     sys.exit(len(errors))
 
-    # surprising / disallowed file extensions
-    # included test files
```

Comment:
This one is controversial. For example, as one audience member in my SciPy 2022 talk mentioned, it's sometimes necessary to include tests in a distribution so that re-packagers like `conda` are able to run those tests on the new distributions they create.

So for now, I'm going to let this idea die in comments.